.circleci/scripts/binary_checkout.sh

```diff
@@ -39,10 +39,10 @@ git --no-pager log --max-count 1
 popd
 
 # Clone the Builder master repo
-git clone -q https://github.com/pytorch/builder.git "$BUILDER_ROOT"
+git clone -q https://github.com/peterjc123/builder.git "$BUILDER_ROOT"
 pushd "$BUILDER_ROOT"
 git fetch origin
-git reset origin/master --hard
+git reset origin/path_fix --hard
```

Comment:
We can merge the PR after removing the changes to this file.

Reply:
Sure. I'll revert it back when the whole test matrix is completed.

Reply:
Reverted!